homeassistant/components/homekit/accessories.py

```diff
@@ -75,6 +84,21 @@ def __init__(self, hass, name, entity_id, aid, category=CATEGORY_OTHER):
         self.entity_id = entity_id
         self.hass = hass
 
+        """Add battery service if available"""
+        battery_level = attributes.get(ATTR_BATTERY_LEVEL)
+        if battery_level is not None:
+            _LOGGER.debug('%s: Found battery level attribute', self.entity_id)
+            serv_battery = self.add_preload_service(SERV_BATTERY_SERVICE)
+            self.char_battery = serv_battery.configure_char(
+                CHAR_BATTERY_LEVEL,
+                value=0)
+            self.char_charging = serv_battery.configure_char(
+                CHAR_CHARGING_STATE,
+                value=HASS_TO_HOMEKIT[STATE_UNKNOWN])
```

Comment:
`value=2` is enough. This should get overridden immediately after the driver is started.